common/scala/build.gradle

```diff
@@ -59,7 +59,7 @@ dependencies {
     compile ('com.fasterxml.uuid:java-uuid-generator:3.1.3')
     compile 'com.github.ben-manes.caffeine:caffeine:2.6.2'
     compile 'com.google.code.findbugs:jsr305:3.0.2'
-    compile 'io.fabric8:kubernetes-client:4.0.3'
+    compile 'io.fabric8:kubernetes-client:4.4.0'
```

Comment:
4.4.2 is the latest point release on the 4.4.x line.  Should we be picking up that instead?

Reply:
Missed that there were new point releases. Done now